package.json

```diff
@@ -31,6 +31,7 @@
     "sass": "^1.37.5",
     "typescript": "^4.3.5",
     "uuid": "^7.0.2",
+    "uuidv4": "^6.2.12",
```

Comment:
Remove this package since we already have the uuid package. (See next comment)

Reply:
missed that. Thanks